doc/usecases/record_elektra/UC_ansible-libelektra_merge_strategies.md

```diff
@@ -0,0 +1,18 @@
+# Use Case: Select merge strategy in ansible-libelektra
+
+## Summary
+
+- **Title:** Select merge-strategy in ansible-libelektra
+- **Scope:** ansible-libelektra
+- **Level:** User goal
+- **Actors:** User (usually sysadmin)
+- **Brief:** It should be possible to specify the merge strategy used, e.g. OURS, THEIRS, ABORT. ansible-libelektra should then use this strategy when merging.
```

Comment:
Please list at least the currently known strategies we want to implement. Ideally the use case gives argumentation when to use what.